src/resource/resource-opening-hours/ResourceOpeningHours.scss

```diff
@@ -1,76 +1,99 @@
 @import 'node_modules/hds-design-tokens/lib/all.scss';
 
-.resource-opening-hours-header {
+.opening-periods-header {
+  --background-color: var(--opening-periods-header-background-color);
```

Comment:
This feels like pretty abstractly named? Is there a chance it might collide with other components? As we don't use module-css to make these unique, we probably have to name then just more uniquely

Reply:
I'll prefix these 👍